### PR TITLE
Utility to reverse the normal orientation of the conditions of a model part

### PR DESCRIPTION
**Pull request by loumalouomega**

### CODE REVIEW

**Comment by philbucher** [comment]
I suggest making the function templated to have it working both for elems and conditions

**Comment by loumalouomega** [comment]
Do you mean for membranes and shells?

**Comment by philbucher** [comment]
potentially yes

the function is very general so I don't see why it should be hardcoded to use conditions

**Comment by loumalouomega** [comment]
Done

File: kratos/python/add_utilities_to_python.cpp [hunk 1]
```diff
@@ -452,6 +452,8 @@ void AddUtilitiesToPython(pybind11::module& m)
     class_<MortarUtilities, typename MortarUtilities::Pointer>(m, "MortarUtilities")
     .def(init<>())
     .def("ComputeNodesMeanNormalModelPart",&MortarUtilities::ComputeNodesMeanNormalModelPart)
+    .def("InvertNormalElements",&MortarUtilities::InvertNormal<PointerVectorSet<Element, IndexedObject>>)
```
Comment (philbucher) [inline kratos/python/add_utilities_to_python.cpp]: I would suggest the name `InvertElementNormals`, "NormalElements" sounds weird to me

your call though

**Review by philbucher — DISMISSED** [review]

**Comment by jcotela** [comment]
I just wanted to mention that we already have quite a few tools to do normal swapping in Kratos (just looking for "Swap" in the core I found a method in TetrahedralMeshOrientationCheck and another in NormalCalculationUtils). I like your implementation, since it is more general and, as @philbucher mentions, might also be useful for shell and membrene elements, but I think it makes sense to consider deprecating some of the old ones if we go ahead with this.

**Comment by loumalouomega** [comment]
Some time ago it was mentioned to unify the normal computation, but not yet done

**Comment by loumalouomega** [comment]
Can anyone approve?

**Review by rubenzorrilla — APPROVED** [review]

**Comment by pooyan-dadvand** [comment]
I just wanted to make emphasis on what @jcotela is mentioning in this PR is not an small issue to be passed by and proceed with approving and merging the PR. 

@loumalouomega your comment related to unifying the normal calculation is not relevant here and in your position I would really tried to check the existing swapping and improve them instead of adding a new one without even a brief explanation of why you need another one.

**Comment by loumalouomega** [comment]
Yes, you are right, I hadn't thought about to improve the current one, sorry. The current one is specific for tetrahedrical meshes, but the one I created was general for any kind of  geometry, that's why wasn't fitted for me, because I use hexahedrical and 2D meshes additionally to the tetrahedrical.

**Comment by loumalouomega** [comment]
I will create an issue to discuss how to unify the utilities

**Pull request merged**